src/main/java/org/cristalise/kernel/persistency/outcome/Outcome.java

```diff
@@ -407,10 +407,10 @@ public String getFieldByXPath(String xpath) throws XPathExpressionException, Inv
      * Determines if the NodeList is actually a single field, an element with text data only
      *
      * @param elements NodeList
-     * @return if the NodeList has a single field or not
+     * @return if the NodeList has a single field of type ELEMENT_NODE or not
```

Comment:
Replace with `@return true if the NodeList has a single field of type ELEMENT_NODE`